src/platform_impl/linux/mod.rs

```diff
@@ -0,0 +1,110 @@
+// Copyright 2022-2022 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+mod icon;
+mod menu;
+mod tray;
+
+pub(crate) use icon::PlatformIcon;
+use tray::Tray;
+
+use crate::{icon::Icon, TrayIconAttributes, TrayIconId};
+
+pub struct TrayIcon {
+    tray_handle: ksni::Handle<Tray>,
+}
+
+impl TrayIcon {
+    pub fn new(id: TrayIconId, attrs: TrayIconAttributes) -> crate::Result<Self> {
+        let icon = attrs.icon.map(|icon| icon.inner.into());
+
+        let title = title_or_pkg_name(attrs.title.unwrap_or_default());
```

Comment:
I would leave this up to the user to specify, we should default to an empty string, also `env!("CARGO_PKG_NAME")` will always be `tray-icon` not the consuming crate